src/test/java/org/encorapedia/pages/EncorapediaBasePage.java

```diff
@@ -0,0 +1,29 @@
+package org.encorapedia.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class EncorapediaBasePage {
+
+    WebDriver driver;
+
+    private By homeLink = By.linkText("Home");
+    private By inventoryLink = By.linkText("Inventory");
+    private By reportsLink = By.linkText("Reports");
+
+    public EncorapediaBasePage(WebDriver driver) {
+        this.driver = driver;
+    }
+    public void clickOnReports() {
+        driver.findElement(reportsLink).click();
+    }
+    public By getHomeLink() {
```

Comment:
Instead of return a By consider return the WebElement and/or include some actions like clickHomeLink...